xclim/indices/_conversion.py

```diff
@@ -116,6 +124,74 @@ def humidex(
     return out
 
 
+@declare_units(tasmax="[temperature]", hurs="[]")
+def heat_index(
+    tasmax: xr.DataArray,
+    hurs: xr.DataArray,
+    freq: str = "MS",
+) -> xr.DataArray:
+    r"""Daily heat index.
+
+    Perceived temperature after relative humidity is taken into account. The
+    index is only valid for temperatures above 20C.
+
+    Parameters
+    ----------
+    tasmax : xr.DataArray
+      Maximum daily temperature.
+    hurs : xr.DataArray
+      Relative humidity.
+    freq : str
+      Resampling frequency.
+
+    Returns
+    -------
+    xr.DataArray, [time][temperature]
+      Heat index for days with temperature above 20C.
+
+    References
+    ----------
+    .. [blazejczyk2012] Blazejczyk, K., Epstein, Y., Jendritzky, G., Staiger,
+    H., & Tinz, B. (2012). Comparison of UTCI to selected thermal indices.
+    International journal of biometeorology, 56(3), 515-535.
```

Comment:
```suggestion
    .. [blazejczyk2012] Blazejczyk, K., Epstein, Y., Jendritzky, G., Staiger, H., & Tinz, B. (2012). Comparison of UTCI to selected thermal indices. International journal of biometeorology, 56(3), 515-535.
```
I know its unelegant to have long lines, but Sphinx/rst won't parse it correctly otherwise.